### ISSUE TITLE
Feature request: chronological list of events

### ISSUE THREAD

**Issue by robin13** [SamAmco/track-and-graph#190]
It would be nice to be able to list all of the events tracked (select time period like last day, week, month, custom time range). This would be useful for example at the end of the day to review everything tracked today to see if anything was forgotten, or to look at a particular day last week for a reminder of what happened over all topics.

**Comment by SamAmco** [comment]
Yeah this is a nice idea. I'll give it some thought. Will probably take me a long time to get round to it though.
